src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -43,7 +43,7 @@ public void Dispose()
         {
             foreach (var thread in Threads)
             {
-                thread.Stop(TimeSpan.FromSeconds(2.5));
+                thread.Stop(TimeSpan.FromSeconds(1.0));
```

Comment:
In use the number gets multiplied by 4 as it moves through the various `_thread.Join` so `2.5` works out at `10` secs. Will divide internally to maintain meaning?